scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,298 @@
+        threshold = index_array.argmax()  # find the first occurrence of True
+        img[img <= threshold] = 0  # set 99% of px with the lowest values to 0
+        img[img > threshold] = 1  # set the remaining 1% to 1
+
+        kernel = cv2.getStructuringElement(cv2.MORPH_RECT, (3, 3))
+        # get rid of noise
+        img_open = cv2.morphologyEx(img, cv2.MORPH_OPEN, kernel, iterations=1)
+        # fill holes in bright square
+        img_close = cv2.morphologyEx(img_open, cv2.MORPH_CLOSE, kernel, iterations=2)
+        images[num_img] = img_close
+
+        # find connected component in thresholded image
+        img_cc, num_labels = skimage.measure.label(img_close, background=0, connectivity=1, return_num=True)
+
+        # there should be only one label in num_labels
+        if num_labels != 1:
+            logging.info("There should be only one active area in the image. However, found {:f} labelled"
+                         "objects.".format(num_labels))
+        # Labelled object should have the size of about 1% of total image in px. Otherwise active area cut of or
+        # outside of the scanning area e.g. due to too much rotation of the mppc.
+        # Use a threshold of 90% of the expected size to allow for variation due to dilation/erosion operation.
+        active_area_found = False
+        # find the connected component
+        # Note: num_labels does not include any background values
+        for label in range(num_labels):
+            # check that the size of the connected component is at least 10% of the size
+            # of the active area (about 1% of the total area)
+            # If not, it is either noise or only part of the active area was captured due to too much rotation
+            # (active area a edge and thus cut-off).
+            if len(img_cc[img_cc == label + 1]) < 0.1 * 0.01 * cell_size_y * cell_size_x:  # as 0 is background
+                # mark connected component as background as it cannot be the active area and
+                # was missed in the other pre-processing steps
+                img_cc[img_cc == label + 1] = 0
+            else:
+                active_area_found = True
+
+        if active_area_found:
+            images[num_img] = img_cc
+        else:
+            raise ValueError("Active area in individual cell image is cut off or not existing. Please check size of"
+                             "scanned area and the coarse alignment of rotation and offset for the mppc detector.")
+
+    return images[0:4], images[4]
+
+
+def get_mppc_rotation(images):
+    """
+    Calculates the rotation of the MPPC detector in space. MPPC detector should be horizontally positioned.
+    The rotation angle is calculated based on the 4 corner cell images by evaluating trigonometric quantities.
+    :param images: (list of 4 ndarrays) Preprocessed corner cell images.
+    :returns: (float) The angle that should be applied to the mppc stage. This is the opposite (negative)
```

Comment:
I'd suggest to return the value in radians, as everywhere else in Odemis. For the *display*, you can convert it to degree.